packages/create-package/create-package.js

```diff
@@ -86,43 +97,35 @@ const createWebcomponentsPackage = async () => {
 		name = response.name;
 	}
 
-	// Add TypeScript support
-	response = await prompts({
-		type: "select",
-		name: "language",
-		message: "Support TypeScript:",
-		choices: [
-			{
-				title: "JavaScript",
-				value: "js",
-			},
-			{
-				title: "TypeScript",
-				value: "ts",
-			},
-		]
-	});
-	const typescript = response.language === "ts";
-
-	// Get the port
-	response = await prompts({
-		type: "text",
-		name: "port",
-		message: "Dev server port:",
-		validate: isPortValid,
```

Comment:
just marking, not to forget - **isPortValid** is not used anymore- we can remove it